### PR TITLE
chore(litmus-portal): Adding Makefile and CI setup for CircleCi

### PR DESCRIPTION
**Pull request by gdsoumya**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Checklist**

* [ ] Does this PR have a corresponding GitHub issue?
* [ ] Have you included relevant README for the chaoslib/experiment with details?
* [ ] Have you added debug messages where necessary? 
* [ ] Have you added task comments where necessary? 
* [ ] Have you tested the changes for possible failure conditions?
* [ ] Have you provided the positive & negative test logs for the litmusbook execution?
* [ ] Does the litmusbook ensure idempotency of cluster state?, i.e., is cluster restored to original state?
* [ ] Have you used non-shell/command modules for Kubernetes tasks?
* [ ] Have you (jinja) templatized custom scripts that is run by the litmusbook, if any? 
* [ ] Have you (jinja) templatized Kubernetes deployment manifests used by the litmusbook, if any?
* [ ] Have you reused/created util functions instead of repeating tasks in the litmusbook?
* [ ] Do the artifacts follow the appropriate directory structure? 
* [ ] Have you isolated storage (eg: OpenEBS) specific implementations, checks? 
* [ ] Have you isolated platform (eg: baremetal kubeadm/openshift/aws/gcloud) specific implementations, checks?  
* [ ] Are the ansible facts well defined? Is the scope explicitly set for playbook & included utils?
* [ ] Have you ensured minimum/careful usage of shell utilities (awk, grep, sed, cut, xargs etc.,)?
* [ ] Can the limtusbook be executed both from within & outside a container (configurable paths, no hardcode)?
* [ ] Can you suggest the minimal resource requirements for the litmusbook execution?
* [ ] Does the litmusbook job artifact carry comments/default options/range for the ENV tunables?
* [ ] Has the litmusbooks been linted? 

**Special notes for your reviewer**:

### CODE REVIEW

File: litmus-portal/Makefile [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+		fi;
+
+.PHONY: build
+build: litmus-portal-build
+
+litmus-portal-build:
+	@echo "------------------"
+	@echo "--> Build frontend and backend images" 
+	@echo "------------------"
+	docker build ./frontend -f frontend/Dockerfile -t gdsoumya/litmusportal-frontend:ci
```
Comment (rajdas98) [inline litmus-portal/Makefile]: gdsoumya/litmusportal-frontend:ci --> litmuschaos/litmusportal-frontend:ci

Comment (rajdas98) [reply litmus-portal/Makefile]: change it in all the places

File: litmus-portal/Makefile [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+litmus-portal-check:
+	@echo "------------------"
+	@echo "--> Check litmus-portal frontend lint"
+	@echo "------------------"
+	cd frontend && npm run lint
+	@echo "------------------"
+	@echo "--> Check litmus-portal backend [gofmt]"
+	@echo "------------------"
+	@fmtRes=$$(gofmt -d backend/graphql-server/*); \
```
Comment (rajdas98) [inline litmus-portal/Makefile:66]: gofmt --> go fmt

File: litmus-portal/Makefile (hunk 1 above)
Comment (gdsoumya) [reply litmus-portal/Makefile]: Made a push with the changes

File: litmus-portal/Makefile (hunk 2 above)
Comment (gdsoumya) [reply litmus-portal/Makefile:66]: Both do almost the same task, go fmt -> gofmt -l -w

**Pull request merged**